src/connectors/trezor_subprovider.js

```diff
@@ -0,0 +1,65 @@
+import { TrezorSubprovider as TrezorSubprovider0x } from '@0x/subproviders/lib/src/subproviders/trezor'
+import web3Utils from 'web3-utils'
+import { getChainIdFromSignedV, buildTransactionForChain, hexToPaddedBuffer } from './utils'
+
+export class TrezorSubprovider extends TrezorSubprovider0x {
+  chainId
+
+  constructor({ chainId, trezorConnectClientApi }) {
+    super({
+      trezorConnectClientApi,
+      networkId: chainId
+    })
+
+    this.chainId = chainId
+  }
+
+  async signTransactionAsync(txData) {
+    if (txData.from === undefined || !web3Utils.isAddress(txData.from)) {
+      throw new Error('Invalid address')
+    }
+    txData.value = txData.value ? txData.value : '0x0'
```

Comment:
this line and the next few could be simplified to use an `||` instead of a ternary like
```
txData.value = txData.value || '0x0'
```